framework/db/Connection.php

```diff
@@ -618,9 +624,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = Yii::createObject([
```

Comment:
From the docs: `Because Yii uses Yii::createObject() in most of its core code to create new objects, this means you can customize the objects globally by dealing with Yii::$container.`

Also you are doing queries which generally take several ms, I don't think the "overhead" of some function calls is relevant here.